src/data/timer-content-loaded.js

```diff
@@ -0,0 +1,8 @@
+const listener = document.addEventListener('pulse-send-pagemonitor', evt => {
+  self.port.emit('timer-content-loaded', evt.detail.timerContentLoaded);
+  document.removeEventListener('pulse-send-pagemonitor', listener);
+});
+
+var sendEvent = document.createEvent('CustomEvent');
```

Comment:
Similarly, not sure if we want to convert all these `var` to `const` below.

Relevant ESLint rules are `no-var` and `prefer-const`, if we want to add them to our ESLint config.